src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -1157,6 +1158,19 @@ void buildHearingPartiesDetails_unavailabilityRanges() throws InvalidMappingExce
         assertThat(unavailabilityRange.getUnavailableToDate()).isEqualTo(end);
     }
 
+    @DisplayName("When the start and end date ranges are both null, then return null")
```

Comment:
Is it worth covering when end date is null but start is fine or an invalid format on either dates as well?

Reply:
added 2 extra tests for this as well as a refactor for commons